igvc_platform/src/tests/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 #add_rostest_gtest(TestJoystickDriver test/test_joystick_driver.test test_joystick_driver.cpp)
+#add_dependencies(TestJoystickDriver igvc_msgs_gencpp)
```

Comment:
I thought this got fixed? am I wrong?

Reply:
I've changed the test so that it works for this implementation on my computer, but I still haven't figured out why it's not receiving messages on the CI server, which is why I commented it out.